src/js/components/Carousel/Carousel.js

```diff
@@ -143,9 +149,13 @@ class Carousel extends Component {
           >
             <Button
               fill="vertical"
-              icon={<PreviousIcon />}
+              icon={
+                <PreviousIcon color={PreviousIconDisabled
```

Comment:
I'm wondering why we don't use `theme.global.control.disabled` here.

Reply:
Actually `theme.global.control.disabled` just adds opacity for the item, but the actual color won't be changed, and my idea was to change icon color when it is disabled, let say we use green color for those controls, but when control is disabled, we want to set a grey color for this control